README.md

```diff
@@ -44,3 +44,17 @@ Create a kubernetes manifest for a pod which will containa ToDo app container:
 1. `README.md` should have commands to apply all the changes
 1. `README.md` should have instructuions on how to validate the changes
 1. Create PR with your changes and attach it for validation on a platform.
+
+## Apply Configurations
+$kubectl apply -f configMap.yml -n todoapp
```

Comment:
Namespace is already specified inside manifests, no need to use the -n flag.
But before the deployment of resource manifests namespace has to be created.
Add command to create namespace.